src/DynamoCoreWpf/Views/Core/DynamoView.xaml

```diff
@@ -334,9 +337,18 @@
                             </MenuItem>
                         </MenuItem>
                         <MenuItem Header="{x:Static p:Resources.DynamoViewFileMenuOpen}"
-                                  Command="{Binding ShowOpenDialogAndOpenResultCommand}"
-                                  Name="openButton"
-                                  InputGestureText="Ctrl + O">
+                                  Name="openButton">
+                            <MenuItem Header="{x:Static p:Resources.DynamoViewFileMenuOpenFile}"
+                                      Name="openFileMenuItem"
+                                      Command="{Binding ShowOpenDialogAndOpenResultCommand}"
+                                      InputGestureText="Ctrl + O">
+                            </MenuItem>
+                            <MenuItem Header="{x:Static p:Resources.DynamoViewFileMenuOpenTemplate}"
+                                      Name="openTemplateMenuItem"
+                                      Command="{Binding ShowOpenTemplateDialogCommand}"
+                                      CommandParameter="Template"
```

Comment:
@zeusongit I decided to use command param so we can distinguish if Dynamo is opening a template so we can reuse more file open code